manifests/charts/base/files/gen-istio-cluster.yaml

```diff
@@ -4969,7 +4952,26 @@ spec:
     storage: true
 ---
 
-
+---
+# Source: base/templates/serviceaccount.yaml
```

Comment:
I thought these service accounts need to be renamed/moved

Reply:
I see that they will be moved in a later PR https://github.com/istio/istio/pull/23194